src/App.js

```diff
@@ -38,6 +38,12 @@ function App() {
   };
 
   const handleGenerateClick = (numOfWeek) => {
+    if (WEEKLY_TASKS.length !== MEMBERS.length) {
+      throw new Error(
+        `WEEKLY_TASKS and MEMBERS array length are not matching to each other`
```

Comment:
Suggest use string literal instead, what if we change the name of the array in the future?